com.blockstream.Green.yml

```diff
@@ -0,0 +1,38 @@
+app-id: com.blockstream.Green
+runtime: org.kde.Platform
+runtime-version: '6.4'
+sdk: org.kde.Sdk
+command: green
+
+finish-args:
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=ipc
+  - --talk-name=org.freedesktop.portal.Background
```

Comment:
I do believe this is unecessary

Reply:
Which part?

Reply:
talk-name that's the line I commented on

Reply:
You're right. Tested without it. I've removed it from the manifest.